.github/workflows/ci.yml

```diff
@@ -0,0 +1,95 @@
+# Copied from: https://github.com/ScopeLift/foundry-template/blob/main/.github/workflows/ci.yml
```

Comment:
Nit, but we can delete this comment IMO. We'll probably adjust the file a bit (e.g. slither preferences, etc) to tailor to the project as things go on, so this may diverge anyway. Feel free to resolve this comment yourself.